### PR TITLE
Update to use opensearch/ESv7, for now using a local/replaced querydsl

### PR DESCRIPTION
**Pull request by ianmcorvidae**
(will update querydsl to real version when merges happen in that repository, see cyverse-de/querydsl#4 )

Ignoring SSL certs is required for now/for testing using localhost. We can possibly remove it later once we have an actual deployment.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,18 +2,18 @@ module github.com/cyverse-de/search
 
 go 1.16
 
+replace github.com/cyverse-de/querydsl => ../querydsl
```
Comment (slr71) [inline go.mod]: This replacement will have to be removed before this can be merged.

**Review by slr71 — APPROVED** [review]
There was a replacement left in go.mod after testing, I didn't see any other problems, though. 👍

File: go.mod (hunk 1 above)
Comment (ianmcorvidae) [reply go.mod]: Yup, I figured I'd merge the querydsl part first and then update this!

**Review by psarando — APPROVED** [review]
LGTM 👍

**Pull request merged**